packages/store/src/module.ts

```diff
@@ -36,8 +36,10 @@ export class NgxsRootModule {
       stateStream.next({ ...cur, ...results.defaults });
     }
 
-    factory.connectActionHandlers(results.states);
```

Comment:
This parameter was in place because it will be required when we do Action Pipes. We can bring it back when needed ;-)